src/integrationTest/java/uk/gov/hmcts/reform/prl/controllers/ManageOrderControllerIntegrationTest.java

```diff
@@ -114,4 +119,17 @@ public void testOrdersCreateUploadEndpoint() throws Exception {
         assertEquals(HttpStatus.SC_OK, httpResponse.getStatusLine().getStatusCode());
     }
 
+    @Test
+    public void testValidateAddressErrorMessageEndpoint() throws Exception {
+        String requestBody = ResourceLoader.loadJson(VALID_REQUEST_RESPONDENT_LIP_WITH_ADDRESS);
+        HttpPost httpPost = new HttpPost(serviceUrl + addressValidationEndpoint);
+        httpPost.addHeader("Content-Type", MediaType.APPLICATION_JSON_VALUE);
+        httpPost.addHeader(AUTHORIZATION, "Bearer testauth");
+        httpPost.addHeader("serviceAuthorization", "s2sToken");
+        StringEntity body = new StringEntity(requestBody);
+        httpPost.setEntity(body);
+        HttpResponse httpResponse = HttpClientBuilder.create().build().execute(httpPost);
+        assertEquals(HttpStatus.SC_OK, httpResponse.getStatusLine().getStatusCode());
```

Comment:
can you please assert the fields rather than status code as agreed before.

Reply:
@shashisk97 added assertions for fields.